hosts_example.yml

```diff
@@ -165,6 +165,63 @@ all:
     # ksql:
     #   properties:
     #     key: val
+    
+    ## Disabaling/Enabling Confluent Metrics Reports and Metrics Interceptors
+    #
+    # Disable Metrics Reports for Confluent Server/Kafka Broker
+    # kafka_broker_metrics:
+    #   enabled: false #Default is true
+    # 
+    # Disable Metrics Interceptors for Kafka Connect and KSqlDB
+    # kafka_connect_metrics:
+    #   enabled: false #Default is true
+    # ksql_metrics:
+    #   enabled: false #Default is true
+    
+    ## Providing custom configurations for Confluent Metrics Reports and Metrics Interceptors
+    #
+    # The configurations for reporters/interceptors are just like those for defining custom listeners.
+    # 
+    # Configuration for Metrics Reporters
+    # kafka_broker_metrics:
+    #   listener: 
+    #     bootstrap_servers: broker1:9092,broker2:9092,broker3:9092
```

Comment:
did this var get moved?

Reply:
Need to update the docs, forgot about that. Will do when I get a moment

Reply:
Docs should be updated now to reflect the changes.